test/agentchat/contrib/vectordb/test_pgvectordb.py

```diff
@@ -48,34 +51,49 @@ def test_pgvector():
     # test_insert_docs
     docs = [{"content": "doc1", "id": "1"}, {"content": "doc2", "id": "2"}, {"content": "doc3", "id": "3"}]
     db.insert_docs(docs, collection_name, upsert=False)
-    res = db.get_collection(collection_name).get(["1", "2"])
-    assert res["documents"] == ["doc1", "doc2"]
+    res = db.get_collection(collection_name).get(ids=["1", "2"])
+    final_results = []
+    for result in res:
+        final_results.append(result.get("content"))
+    assert final_results == ["doc1", "doc2"]
 
     # test_update_docs
     docs = [{"content": "doc11", "id": "1"}, {"content": "doc2", "id": "2"}, {"content": "doc3", "id": "3"}]
     db.update_docs(docs, collection_name)
     res = db.get_collection(collection_name).get(["1", "2"])
-    assert res["documents"] == ["doc11", "doc2"]
+    final_results = []
+    for result in res:
+        final_results.append(result.get("content"))
+    assert final_results == ["doc11", "doc2"]
 
     # test_delete_docs
     ids = ["1"]
     collection_name = "test_collection"
     db.delete_docs(ids, collection_name)
     res = db.get_collection(collection_name).get(ids)
-    assert res["documents"] == []
+    final_results = []
+    for result in res:
+        final_results.append(result.get("content"))
+    assert final_results == []
 
     # test_retrieve_docs
     queries = ["doc2", "doc3"]
     collection_name = "test_collection"
     res = db.retrieve_docs(queries, collection_name)
-    assert [[r[0]["id"] for r in rr] for rr in res] == [["2", "3"], ["3", "2"]]
+    final_results = []
+    for result, dtype in res[0]:
+        final_results.append(result.get("id").strip())
+    assert final_results == ["2", "3", "3", "2"]
```

Comment:
The format of `res` doesn't meet the protocol.

Reply:
What is the expectation? Are you looking for an exact data format? 

assert final_results == ["3", "2"]?

Reply:
Resolved